app/controllers/api/v3/provider_suggestions_controller.rb

```diff
@@ -0,0 +1,27 @@
+module API
+  module V3
+    class ProviderSuggestionsController < API::V3::ApplicationController
+      before_action :build_recruitment_cycle
+
+      def index
```

Comment:
Potentially scope to have a shared action made available via a concern so that V2 and V3 could access? V2 is slightly different however in that it does authentication ...

Reply:
I don't think this bit of duplication is too bad. We're changing the number of results returned and the v2 is the suggest action.

Reply:
I would still be tempted to use a service ... I'm not an advocate for services-for-absolutely-everything, but search does seem like a good spot to use a service, even if the logic is minimal. We can address this later though.